MATLAB/Utilities/VarianCBCT/ROIMask.m

```diff
@@ -0,0 +1,34 @@
+function Result = ROIMask(img, dia)
+%% Cylindrical Mask to Input image matrix
+% Date: 2015-10-22
+% Author: Yi Du (yi.du@hotmail.com)
+
+[Length, Width, Height] = size(img);
+
+if(Length~=Width)
+    error('Length is not equal to Width');
+end
+Length = Width;
```

Comment:
I coded this function years ago when I was fresh new in CT. I don't know why I put it like this. : )
BTW, I didn't use the cropCBCT in TIGRE/Utilities, because cropCBCT doesn't work correctly for half fan scan. The algorithm therein needs revision.

Reply:
No worries! I have sat down and modified this file, works know with a displaced image too and any image size.  Sometimes I am just leaving notes so I know what to modify ;)